openpype/hosts/nuke/api/lib.py

```diff
@@ -3116,3 +3116,14 @@ def get_viewer_config_from_string(input_string):
             ).format(input_string))
 
     return (display, viewer)
+
+
+def get_app_version_info():
+    """ Return dict of Nuke's version semantic info"""
+    dot_split = nuke.NUKE_VERSION_STRING.split(".")
+    v_spit = dot_split[1].split("v")
+    return {
```

Comment:
I think this could be achieved with:
```py
return {
    "major": int(nuke.NUKE_VERSION_MAJOR),
    "minor": int(nuke.NUKE_VERSION_MINOR),
    "path": int(nuke.NUKE_VERSION_RELEASE), # not entirely sure about this one, but I believe it should be there, or similarly named
}```

Reply:
Ha-ha, yeah, you are right. I haven't come across those in my quick google search ;) Thanks